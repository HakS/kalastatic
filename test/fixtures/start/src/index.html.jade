---
title: Jade
youAreUsingJade: true
pretty: true
---
doctype html
html(lang="en")
  head
    title= title
  body
    h1 Jade - node template engines
  script(src="scripts.js")
