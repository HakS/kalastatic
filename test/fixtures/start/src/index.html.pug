---
title: Hello World!
pretty: true
---
doctype html
html(lang="en")
  head
    title= title
  body
    h1 KalaStatic BrowserSync Watch/Start
    p This is built from the "test/fixtures/start" directory.
    ol
      li Run npm start
      li Open up test/fixtures/start/src/index.html.pug
      li Change something
      li See it reflet in the browser after it's built
      li WOW!
