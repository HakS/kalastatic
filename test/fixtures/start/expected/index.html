<!DOCTYPE html>
<html lang="en">
  <head>
    <title>Hello World!</title>
  </head>
  <body>
    <h1>KalaStatic BrowserSync Watch/Start</h1>
    <p>This is built from the "test/fixtures/start" directory.</p>
    <ol>
      <li>Run npm start</li>
      <li>Open up test/fixtures/start/src/index.html.pug</li>
      <li>Change something</li>
      <li>See it reflet in the browser after it's built</li>
      <li>WOW!</li>
    </ol>
  </body>
</html>
